src/CommandExecutor.ts

```diff
@@ -51,6 +51,12 @@ class BuiltInCommandExecutionStrategy extends CommandExecutionStrategy {
 class SystemFileExecutionStrategy extends CommandExecutionStrategy {
     startExecution() {
         return new Promise((resolve, reject) => {
+            if (process.platform === 'win32') {
+                this.args.unshift(this.command);
+                this.args = ['/s', '/c', this.args.join(' ')];
```

Comment:
Have you tried executing a command with multiple parameters? I'm not sure it works as expected.